mobile_verifier/src/reward_shares.rs

```diff
@@ -3,17 +3,132 @@ use crate::{
     speedtests::{Average, SpeedtestAverages},
 };
 use chrono::{DateTime, Duration, TimeZone, Utc};
-use file_store::traits::TimestampEncode;
+use file_store::{mobile_transfer::ValidDataTransferSession, traits::TimestampEncode};
+use futures::{Stream, StreamExt};
 use helium_crypto::PublicKeyBinary;
 use helium_proto::services::{
     follower::{self, follower_gateway_resp_v1::Result as GatewayResult, FollowerGatewayReqV1},
     poc_mobile as proto, Channel,
 };
-use lazy_static::lazy_static;
 use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 use std::collections::HashMap;
 use std::ops::Range;
 
+/// 100M genesis rewards per day
+const GENESIS_REWARDS_PER_DAY: Decimal = dec!(100_000_000_000_000);
```

Comment:
now that you have TOTAL_EMISSIONS_POOL and data transfer is 40% of that, do we need to define GENESIS_REWARDS_PER_DAY ?

Reply:
_GENESIS_ rewards should probably go away or change once we're out of the genesis period